Apps/Playground/node_modules/@babylonjs/react-native/JSCRuntime.cpp

```diff
@@ -0,0 +1,1448 @@
+// NOTE: This file was copied from the JSCRuntime.cpp included in React Native 0.62.1 with modifications in order to:
+//   1. Access the underlying JSC JSGlobalContextRef.
+//   2. Create a custom JSCRuntime with bug fixes and performance improvements.
```

Comment:
Note that latest version of JSCRuntime has some of the perf issues already done.